examples/karpenter/main.tf

```diff
@@ -45,209 +53,231 @@ locals {
   }
 }
 
-#---------------------------------------------------------------
-# EKS Blueprints
-#---------------------------------------------------------------
-
-module "eks_blueprints" {
-  source = "../.."
-
-  cluster_name    = local.name
-  cluster_version = "1.24"
-
-  vpc_id             = module.vpc.vpc_id
-  private_subnet_ids = module.vpc.private_subnets
-
-  #----------------------------------------------------------------------------------------------------------#
-  # Security groups used in this module created by the upstream modules terraform-aws-eks (https://github.com/terraform-aws-modules/terraform-aws-eks).
-  #   Upstream module implemented Security groups based on the best practices doc https://docs.aws.amazon.com/eks/latest/userguide/sec-group-reqs.html.
-  #   So, by default the security groups are restrictive. Users needs to enable rules for specific ports required for App requirement or Add-ons
-  #   See the notes below for each rule used in these examples
-  #----------------------------------------------------------------------------------------------------------#
-  node_security_group_additional_rules = {
-    # Extend node-to-node security group rules. Recommended and required for the Add-ons
-    ingress_self_all = {
-      description = "Node to node all ports/protocols"
-      protocol    = "-1"
-      from_port   = 0
-      to_port     = 0
-      type        = "ingress"
-      self        = true
-    }
-    # Recommended outbound traffic for Node groups
-    egress_all = {
-      description      = "Node all egress"
-      protocol         = "-1"
-      from_port        = 0
-      to_port          = 0
-      type             = "egress"
-      cidr_blocks      = ["0.0.0.0/0"]
-      ipv6_cidr_blocks = ["::/0"]
+################################################################################
+# Cluster
+################################################################################
+
+#tfsec:ignore:aws-eks-enable-control-plane-logging
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "~> 19.5"
+
+  cluster_name                   = local.name
+  cluster_version                = "1.24"
+  cluster_endpoint_public_access = true
```

Comment:
this is a bad example default, from a security point of view, considering many will opy the examples as their deployment reference.

Reply:
all examples, minus the fully private example, have the public endpoint enabled. This was done intentionally to make it easier to get started with interacting with the clusters from the examples as well as for testing from the GitHub workflows. The only change here is that it is now explicitly clear that the public endpoint is being enabled 

Was there some value that was meant to be added by posting this comment?